community/rpm.sh

```diff
@@ -78,10 +78,39 @@ detect_os ()
   echo "Detected operating system as ${os}/${dist}."
 }
 
+add_pgdg ()
+{
+  if [[ "$os" = "fedora" ]]; then
+    suffix="3"
+    family="fedora"
+    family_short="fedora"
+  else
+    suffix="2"
+    family="redhat"
+    family_short="rhel"
+  fi
+
+  if [[ "$os" = "ol" ]]; then
+    pkg_os="oraclelinux"
+  else
+    pkg_os="$os"
+  fi
+
+  if [[ "$(arch)" = "x86_64" ]]; then
+    arch="x86_64"
+  else
+    # sometimes reports as i686
+    arch="i386"
+  fi
+
+  yum install -y https://download.postgresql.org/pub/repos/yum/9.5/$family/$family_short-$dist-$arch/pgdg-${pkg_os}95-9.5-$suffix.noarch.rpm
```

Comment:
This is _super_ long. Maybe break across several lines, building it up via appends?